example/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: dredd
+
+hooks/hooks:
+	cd hooks/; go build; cd ..;
+
+server:
+	go build -o server
+
+dredd: hooks/hooks server
+	~/Code/dredd/bin/dredd api.apib http://localhost:8080 --language go --server ./server --hookfiles hooks/hooks
```

Comment:
@snikch once it is released tomorrow I will be updating that path to use a local `./node_modules/.bin/dredd`